docs/xdr/features/collect/integrations/network/sonicwall_fw.md

```diff
@@ -26,7 +26,14 @@ to SEKOIA.IO by means of a syslog transport channel.
 
 ### Enable Syslog forwarding for SonicWall 6.5 and 7.X
 
-Log on your SonicWall appliance and follow this [guide](https://www.sonicwall.com/support/knowledge-base/how-can-i-configure-a-syslog-server-on-a-sonicwall-firewall/170505984096810/) to enable syslog forwarding.
+### Disclamer
```

Comment:
I think that this title is redundant with the note below (And it's as the same level as the previous one).
```suggestion
```